From 36705293a3c9fd35cb30f86bd080006d703b1e2b Mon Sep 17 00:00:00 2001
From: Cody <cody@quantifiedcode.com>
Date: Wed, 15 Jun 2016 22:04:26 +0000
Subject: [PATCH] Migrated `%` string formating

---
 web_scraping.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/web_scraping.py b/web_scraping.py
index de26f73c4ba9b976e2ab529af02bc6ad4ac50325..ffd3772073d2379ccd27c6faef1a3e449ec9a6b0 100644
--- a/web_scraping.py
+++ b/web_scraping.py
@@ -326,10 +326,10 @@ def add_download_data_to_csv_file(dest_path, row_index):
             try:
                 with open(txt_file, "r+") as in_txt:
                     read_in = csv.reader(in_txt, delimiter='\t')
-                    print "%s open success!" % (txt_file)
+                    print "{0!s} open success!".format((txt_file))
                     with open(csv_file, CONTINUE_WRITE) as out_csv:
                         write_to = csv.writer(out_csv)
-                        print "%s open success!" % (dest_path)
+                        print "{0!s} open success!".format((dest_path))
                         if is_head_list == 1:
                             # if this is the first time accessing the target file
                             for line in read_in:
@@ -414,7 +414,7 @@ if __name__ == '__main__':
             raise Exception('something wrong with config file')
         if CITATION_LIST_ONLY == 1 and RETRACTION_LIST_ONLY == 0:
             print "CITATION_LIST_ONLY"
-            print "MAX_CITATION_NUMBER %s" % (MAX_CITATION_NUMBER)
+            print "MAX_CITATION_NUMBER {0!s}".format((MAX_CITATION_NUMBER))
             take_out_title_from_retraction_list()
         elif RETRACTION_LIST_ONLY == 1 and CITATION_LIST_ONLY == 0:
             print "RETRACTION_LIST_ONLY"
